tests/test_views.py

```diff
@@ -7,13 +7,20 @@
 import requests
 from rest_framework.test import APIClient
 
+from s3_file_field import _multipart
+
 from .fuzzy import URL_RE, UUID_RE, Re
 
 
 def mb(bytes_size: int) -> int:
     return bytes_size * 2 ** 20
 
 
+@pytest.fixture
+def mock_part_size(monkeypatch):
+    monkeypatch.setattr(_multipart, 'DEFAULT_PART_SIZE', mb(5))
```

Comment:
I'd like to use `pytest-mock` generally, since it's significantly more powerful in other cases, and it's good to use only one mocking library. I realize that wasn't a dependency though.